Sparkle/SUUIBasedUpdateDriver.m

```diff
@@ -114,6 +114,10 @@ - (void)updateAlertFinishedWithChoice:(SUUpdateAlertChoice)choice
     [self.host setObject:nil forUserDefaultsKey:SUSkippedVersionKey];
     switch (choice) {
         case SUInstallUpdateChoice:
+            if (![self verifyHostVolumeWritable]) {
+                return;
```

Comment:
Do I read this correctly that if user presses the Install button, nothing will happen at all?